src/App.css

```diff
@@ -1,3 +1,32 @@
 .App {
-  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 1000px;
+  margin: 0 auto;
+  font-family: monospace;
+  background-color: rgb(230, 199, 142);
+  border: 5px solid black;
+  border-radius: 32px;
+}
+
+.App__list-title {
+  font-weight: 600;
+  font-size: 16px;
+}
+
+.App__form {
+  display: flex;
+  flex-direction: column;
+  padding: 5px;
+  width: 300px;
+  background-color: rgb(245, 222, 179);
+  border: 3px solid black;
+  border-radius: 16px;
+}
+
+.App__input {
```

Comment:
Use SCSS, instead. In the project, scss package already included, so you should use it